### PR TITLE
fix(deps): update package lock json to date

### PR DESCRIPTION
**Pull request by MCatherine1994**
to address the Dependabot security warnings
- @babel/traverse
- postcss
- tough-cookie

Removed the existing package-lock.json under admin, libs and public, did a fresh install to bring package-lock.json up to date. Leave the api part for now, we'll handle it separately. 

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
 - [api](https://fom-45.apps.silver.devops.gov.bc.ca/api)
 - [admin](https://fom-45.apps.silver.devops.gov.bc.ca/admin)
 - [public](https://fom-45.apps.silver.devops.gov.bc.ca/public)

Once merged, code will be promoted and handed off to following workflow run.
 - [Main Merge Workflow](https://github.com/bcgov/nr-fom/actions/workflows/merge-main.yml)

### CODE REVIEW

File: public/package-lock.json [hunk 1]
```diff
@@ -16935,9 +17178,9 @@
       }
     },
     "node_modules/tough-cookie": {
-      "version": "4.1.2",
-      "resolved": "https://registry.npmjs.org/tough-cookie/-/tough-cookie-4.1.2.tgz",
-      "integrity": "sha512-G9fqXWoYFZgTc2z8Q5zaHy/vJMjm+WV0AkAeHxVCQiEB1b+dGvWzFW6QV07cY5jQ5gRkeid2qIkzkxUnmoQZUQ==",
+      "version": "4.1.3",
```
Comment (MCatherine1994) [inline public/package-lock.json:17181]: updated tough-cookie to 4.1.3 https://github.com/bcgov/nr-fom/security/dependabot/127

File: public/package-lock.json [hunk 2]
```diff
@@ -14930,9 +15111,9 @@
       }
     },
     "node_modules/postcss": {
-      "version": "8.4.21",
-      "resolved": "https://registry.npmjs.org/postcss/-/postcss-8.4.21.tgz",
-      "integrity": "sha512-tP7u/Sn/dVxK2NnruI4H9BG+x+Wxz6oeZ1cJ8P6G/PZY0IKk4k/63TDsQf2kQq3+qoJeLm2kIBUNlZe3zgb4Zg==",
+      "version": "8.4.31",
```
Comment (MCatherine1994) [inline public/package-lock.json:15114]: updated postcss to 8.4.31 https://github.com/bcgov/nr-fom/security/dependabot/144

File: libs/package-lock.json [hunk 3]
```diff
@@ -578,48 +641,48 @@
...
       },
       "engines": {
         "node": ">=6.9.0"
       }
     },
     "node_modules/@babel/traverse": {
-      "version": "7.22.5",
-      "resolved": "https://registry.npmjs.org/@babel/traverse/-/traverse-7.22.5.tgz",
-      "integrity": "sha512-7DuIjPgERaNo6r+PZwItpjCZEa5vyw4eJGufeLxrPdBXBoLcCJCIasvK6pK/9DVNrLZTLFhUGqaC6X/PA007TQ==",
+      "version": "7.23.7",
```
Comment (MCatherine1994) [inline libs/package-lock.json:658]: updated @babel/traverse to greater than 7.23.2, https://github.com/bcgov/nr-fom/security/dependabot/146

**Comment by MCatherine1994** [comment]
updated react-devtools-core to v4.28.5 (admin/package-lock.json), https://github.com/bcgov/nr-fom/security/dependabot/149
Screen Shot 2024-01-03 at 2 52 00 PM

**Review by DerekRoberts — APPROVED** [review]

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**